Libraries/LibGUI/GTextEditor.cpp

```diff
@@ -482,11 +484,12 @@ void GTextEditor::undo()
     }
 
     m_undo_stack_index++;
+    on_change();
```

Comment:
It's not safe to call `on_change` directly, since the user of this `GTextEditor` may not have installed a hook.
Instead, call `did_change()`.